### PR TITLE
Issue #86 : Removed unused import os in test_uiserver.py

### LINKED ISSUE #86: Remove unused import in test_uiserver
`test_uiserver.py` has an import for `os`, but doesn't seem to be using that module. Should be pretty straightforward. Just double-check that the `os` module is not referenced in that module, then remove and make sure tests pass.

### PR DESCRIPTION
**Pull request by smcveigh941**
Closes #86

### CODE REVIEW

**Comment by kinow** [comment]
Hi @smcveigh941 ! 

Thanks for your contribution! Could you take a look at the `CONTRIBUTING.md` and, if you are OK with that, add your name to the contributors list, please?

Thanks again!
Bruno

**Comment by smcveigh941** [comment]
Hi @kinow,

I have added to the contribution list now :)

(sorry I had originally submitted the change to the CONTRIBUTING.md file as a separate PR which I have now closed)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -38,6 +38,7 @@ below.
  - Bruno Kinoshita
  - David Sutherland
  - Tim Pillinger
+ - Stephen McVeigh
```
Comment (kinow) [inline CONTRIBUTING.md]: Brilliant!!! Thanks a lto!

**Comment by kinow** [comment]
Tests passed on Travis (change was to one of the tests), one review should do. Merging :tada: 

Thanks again for your contribution @smcveigh941 !
Bruno

**Pull request merged**